build/Common.prod.props

```diff
@@ -20,17 +20,23 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <!-- Optional: Declare that the Repository URL can be published to NuSpec -->
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageTags>Tracing;OpenTelemetry;Management;Monitoring</PackageTags>
+    <PackageIconUrl>https://opentelemetry.io/img/logos/opentracing-icon-color.png</PackageIconUrl>
```

Comment:
Please change to https://opentelemetry.io/img/logos/opentelemetry-icon-color.png